.github/dependabot.yml

```diff
@@ -1,5 +1,6 @@
 
 version: 2
+enable-beta-ecosystems: true
```

Comment:
Is this necessary?  According to the documentation here it doesn't do anything: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#enable-beta-ecosystems-

Reply:
Spoke offline and understand the usage now.